pramen-py/tests/test_metastore/test_metastore.py

```diff
@@ -26,12 +26,56 @@
 from chispa.schema_comparer import SchemasNotEqualError
 from loguru import logger
 from pyhocon import ConfigFactory  # type: ignore
-from pyspark.sql import SparkSession
+from pyspark.sql import DataFrame, SparkSession
 
 from pramen_py import MetastoreReader, MetastoreWriter
 from pramen_py.models import InfoDateSettings, MetastoreTable, TableFormat
 
 
+def test_metastore_get_latest_available_date_for_delta(
+    spark, get_data_stub, tmp_path
+):
+    def save_delta_table(df: DataFrame, path: str) -> None:
+        df.write.partitionBy("info_date").format("delta").mode(
+            "overwrite"
+        ).save(path)
+
+    table_name = "latest_available_date"
+    table_path = (
+        tmp_path / "data_lake" / "example_test_tables" / table_name
+    ).as_posix()
+
+    df_union = get_data_stub.union(
+        spark.createDataFrame(
+            spark.sparkContext.parallelize([(17, 18, d(2022, 11, 2))]),
```

Comment:
Sorry for nitpicking but I think you can create a DataFrame directly from a list of tuples (without using RDD).
```suggestion
            [(17, 18, d(2022, 11, 2))],
```

Reply:
You right, Thanks!